### PR TITLE
Add icse 24

### LINKED ISSUE #52: ICSE'24
Let's add papers from ICSE'24.

### PR DESCRIPTION
**Pull request by fraglantia**
Resolve #52

### CODE REVIEW

File: data/fuzzers.json [hunk 1]
```diff
@@ -6048,6 +6263,32 @@
...
+  {
+    "name": "SpecBCFuzz",
+    "year": "2024",
+    "targets": [
+      "LTL Solver"
+    ],
+    "color": "blackbox",
+    "references": [
+      "STORM",
+      "AFL"
```
Comment (haeunlee31) [inline data/fuzzers.json:6275]: I couldn't find why AFL would be a reference for SpecBCFuzz from the paper, maybe STORM is a better fit?

Comment (fraglantia) [reply data/fuzzers.json:6275]: Ah I missed STORM from the paper. I agree STORM is a better fit.

File: data/fuzzers.json [hunk 2]
```diff
@@ -1073,6 +1073,29 @@
       "https://dl.acm.org/doi/pdf/10.1145/3611643.3616332"
     ]
   },
+  {
+    "name": "ConfigFuzz",
```
Comment (haeunlee31) [inline data/fuzzers.json:1077]: I'm not sure if ConfigFuzz meets the inclusion criteria for our survey. Is there another reference for ECFuzz other than ConfigFuzz?

Comment (sangkilc) [reply data/fuzzers.json:1077]: I think we don't need to be super strict about adding more fuzzers to our database at this point. Let's just add it.

**Pull request merged**